templates/home.html

```diff
@@ -1,6 +1,42 @@
 {% extends 'base.html' %}
 {% load static %}
 
+{% block css %}
+<link rel="stylesheet" href="{% static 'css/home.css' %}">
+{% endblock %}
+
 {% block content %}
-<h1>Home</h1>
-{% endblock content %}
\ No newline at end of file
+<div class="jumbotron" id="planner-container">
+    <h1>Grocery Planner</h1>
+    <div class="card border-primary mb-3" id="card_for_image">
+        <div class="card-body">
+            <a href='planner'>
+            <img class="image_in_card" src="{% static 'images/home_planner_produce.jpg' %}">
+            </a>
+            <figcaption>Image Credit: Wholesome Culture</figcaption>
+        </div>
+    </div>
+    <hr class="my-4">
+    <p id='planner-description'>Plan your grocery list by choosing from your favorite meals.</p>
+    <p class="lead">
+        <a class="btn btn-primary btn-lg" href="planner" role="button">Start Planning</a>
+    </p>
+</div>
+
+<div class="jumbotron" id="recipes-container">
+    <h1>Recipes</h1>
+    <div class="card border-primary mb-3" id="card_for_image">
```

Comment:
What does mb-3 do?

Reply:
That was included in the bootswatch for the card. Looks like its spacing, setting a default of the margin bottom to 3.

'Bootstrap includes a wide range of shorthand responsive margin and padding utility classes to modify an element’s appearance.'
https://getbootstrap.com/docs/4.0/utilities/spacing/